API/ExecutorInterface.php

```diff
@@ -27,4 +27,13 @@ public function supportedTypes();
      * @throws \Exception
      */
     public function execute(MigrationStep $step);
+
+    /**
+     * Generate a update template
+     *
+     * @param string $identifier
+     * @return string Yaml dump
```

Comment:
also: this should return an array, as we now support json as migration format. We should leave the encoding to either yaml or json to the Command class